packages/drawer/src/react/__specs__/storyshots.spec.js

```diff
@@ -0,0 +1,14 @@
+import initStoryshots, { snapshotWithOptions } from '@storybook/addon-storyshots'
+
+function createNodeMock(element) {
```

Comment:
Since Jest is running in a dom-less environment, it doesn't have dom elements to pass when you have refs in your components. But you can do this to provide mock elements that get passed instead of null.